code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -21,7 +20,13 @@
 	var/skill_required = SKILL_PILOT_TRAINED
 	var/combat_equipment = TRUE
 	var/faction_exclusive //if null all factions can print it
+	var/stackable_ammo = FALSE 	/// Whether the ammo inside this equipment can be directly replenished without needing to uninstall the existing ammo
```

Comment:
https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md#required 

these vars (really the more the merrier but especially these) should be autodoc'd